src/core/remote.js

```diff
@@ -1209,35 +1212,21 @@ Remote.prototype.requestTx = function(options_, callback_) {
  */
 
 Remote.accountRequest = function(command, options_, callback_) {
-  let options, callback = callback_;
+  const options = {callback: callback_};
 
   if (lodash.isPlainObject(options_)) {
-    options = lodash.merge({}, options_);
+    lodash.merge(options, options_);
   } else {
-    log.warn('DEPRECATED: First argument to request constructor should be'
-      + ' an object containing request properties');
-
-    const args = Array.prototype.slice.call(arguments);
-
-    if (lodash.isFunction(lodash.last(args))) {
-      callback = args.pop();
-    }
-
-    options = {
-      account: args.shift(),
-      peer: args.shift(),
-      ledger: args.shift(),
-      limit: args.shift(),
-      marker: args.shift()
-    };
+    lodash.merge(options, makeOptions(
+      command,
+      ['account', 'ledger', 'peer', 'limit', 'marker'],
```

Comment:
Was this change intentional?  It went account/peer/ledger before and now it's account/ledger/peer ?